File: _challenges/6696c2ca763f0712c198dbf7_challenge.js
// NAMEEN:Reduced by two times
// NAMERU:Уменьшение в 2 раза

/*
INSTRUCTIONENSTART
Variable `sum` is created and has a value. Write a command that reduces the existing value of `sum` variable by a factor of 2.

Here is an example that is fundamentally similar to what needs to be done, but adapt it to the requirements of the task. For example, you want to increase the existing value of variable `count` by 3 times. To do this, you can write the command:
```
count = count * 3;
```
INSTRUCTIONENEND
*/

/*
INSTRUCTIONRUSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-7.png)

Нужно создать переменную `sum`, при помощи ключевого слова `let` и присвоить ей значение `20`.

В следующей строке нужно записать команду, уменьшающую имеющееся значение переменной `sum` в 2 раза.

Приведем пример, который принципиально похож на то, что нужно сделать, но адаптируйте его к требованиям задачи.

Например, требуется увеличить имеющееся значение переменной `count` в 3 раза.

Для этого можно записать команду:
```javascript
let count = 10;
count = count * 3;
```
INSTRUCTIONRUEND
*/

/*
HINTENSTART
```javascript
let sum = 20;
sum = sum / 2;
```
HINTENEND
*/

/*
HINTRUSTART
```javascript
let sum = 20;
sum = sum / 2;
```
HINTRUEND
*/

/*
INITIALSOLUTIONSTART
let sum = 20;
INITIALSOLUTIONEND
*/

// SOLUTIONSTART
let sum = 20;
sum = sum / 2;
// SOLUTIONEND


// OPENTESTSSTART
it('variable "sum" exists and has a value', () => {
    expect(sum).not.undefined;
});

it('variable "sum" is not NaN', () => {
    expect(isNaN(sum)).false;
});

it('variable "sum" has the correct value', () => {
    expect(sum).equal(10);
});
it('variable "sum" is of type number', () => {
    expect(sum).a('number');
});
// OPENTESTSEND

// HIDDENTESTSSTART

// HIDDENTESTSEND

File: _challenges/6696c2ca763f0712c198dbf7_challenge.py
// NAMEEN:The unpredictability of the game
// NAMERU:Непредсказуемость игры

/*
INSTRUCTIONENSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-8.png)

A tourist in Las Vegas started a game with some amount of money.

Create a variable `start_money` and assign it a value greater than or equal to `100`.

At the end of the game, he lost some amount of money.

Create a variable `lost_money` and assign it a value between `50` and `80`.

Create a variable `end_money` and assign it the calculated value of the remaining money,

which is the difference between `start_money` and `lost_money`.
INSTRUCTIONENEND
*/

/*
INSTRUCTIONRUSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-8.png)

Турист в Лас-Вегасе начал игру со какой-то суммой денег.

Создайте переменную `start_money` и присвойте ей значение большее либо равное `100`.

В конце игры он проиграл какую-то сумму денег.

Создайте переменную `lost_money` и присвойте ей значение от `50` до `80`.

Создайте переменную `end_money` и присвойте ей вычисленное значение оставшихся денег, как разницу между `start_money` и `lost_money`.
INSTRUCTIONRUEND
*/

/*
HINTENSTART
Here is a similar example:
```python
start_money = 150
lost_money = 60
end_money = start_money - lost_money
```
HINTENEND
*/

/*
HINTRUSTART
Похожий пример:
```python
start_money = 150
lost_money = 60
end_money = start_money - lost_money
```
HINTRUEND
*/

/*
INITIALSOLUTIONSTART

INITIALSOLUTIONEND
*/

// SOLUTIONSTART
start_money = 150
lost_money = 60
end_money = start_money - lost_money
// SOLUTIONEND

// OPENTESTSSTART
it('The "start_money" variable contains a value more or equal 100', () => {
  expect(start_money).gte(100);
});

it('The "lost_money" variable contains a value between 50 and 80', () => {
  expect(lost_money).gte(50).lte(80);
});

it('The "end_money" variable contains the difference between "start_money" and "lost_money"', () => {
  expect(end_money).equal(start_money - lost_money);
// OPENTESTSEND

// HIDDENTESTSSTART

// HIDDENTESTSEND
